doc/import-commitments.md

```diff
@@ -8,7 +8,7 @@ At this point the application can only import commitments.
 ## Importing
 Commitments are imported by the `commitments:import` rake task.
 
-You will need a csv of valid commitments provided by BEIS:
+You will need a csv of valid commitments provided by BEIS. 'RODA indentifier' and 'Commitment value' are mandatory. 'Transaction date' is optional.
```

Comment:
I wonder if we should remove the `Transaction date` from this completely, as it looks like it's not actually being used here - I don't know why I added it to this documentation 🤦

Reply:
also, small typo in `RODA i[n]dentifier` here.